Added scoped access control docs (#34729)
openshift/openshift-docs: operating/manage-user-access/manage-role-based-access-control-3062.adoc

[id="manage-role-based-access-control"]

> = Managing role-based access control in {product-title} 3.0.62 and older

Reviewer: As in my previous comment, I suggest using "RBAC" in the title

include::modules/common-attributes.adoc[]
:context: manage-role-based-access-control

toc::[]

[role="_abstract"]

> {product-title} (RHACS) comes with role-based access control (RBAC) that you can use to configure roles and grant various levels of access to {product-title} to different users.

Reviewer: Maybe "to different users" --> "for different users" so you aren't repeating "to" again? It was sort of hard to read this the first time.

[IMPORTANT]
====

> In the {product-title} 3.63.0, we have added a scoped access control feature.

Reviewer: ```suggestion
In {product-title} 3.63.0, a scoped access control feature has been added.
```

> If you are using {product-title} 3.63.0 or user see xref:../../operating/manage-user-access/manage-role-based-access-control-3630.adoc#manage-rbac-3630[Managing access control in {product-title} 3.63.0 and newer].

Reviewer: ```suggestion
If you are using {product-title} 3.63.0 or newer, see xref:../../operating/manage-user-access/manage-role-based-access-control-3630.adoc#manage-rbac-3630[Managing access control in {product-title} 3.63.0 and newer].
```

====

In {product-title} 3.0.62 and older, different roles govern access to {product-title} resources to prevent unwanted access.

> * _Roles_ are a collection of rules, which are a group of `read` and `write` permissions that user can perform on a set of resources.

Reviewer: "that _a_ user can perform"
